thorn/include/common/geometry.h

```diff
@@ -0,0 +1,16 @@
+#ifndef _ROSE_C_GEOMETRY_H
+#define _ROSE_C_GEOMETRY_H
+
+typedef struct {
+    unsigned int x, y;
+} point_t;
+
+#define POINT(x, y) \
+    (point_t) {     \
+        x, y        \
+    }
+
+void    print_point (point_t p1);
```

Comment:
I'd appreciate a more descriptive name than "print_". We have far too many "print" functions already

Reply:
what else would you suggest?

The reason why I named it print was because we had this distinction between drawing and printing. Drawing the point would draw it to the screen as a pixel. Printing the point means printing the information of the point as text.

Reply:
my issue